setup.py

```diff
@@ -33,7 +39,6 @@
       install_requires=[
           'setuptools',
           'Zope >= 4.0b4',
```

Comment:
```suggestion
          'Zope >= 4.3',
```

Zope 4.3 added support for WebDAV indepenently from ZServer.